src/App.svelte

```diff
@@ -13,7 +13,8 @@
 </script>
 
 <main class="container">
-  <h1>Create your own jigsaw!</h1>
+  <h1>Welcome to our website</h1>
+  <h1>Create your own Jigsaw!</h1>
```

Comment:
From semantic point of view two `h1` tags in a row don't look that good, could you change one of them to `h2`?